elasticdl/python/master/master.py

```diff
@@ -85,8 +104,12 @@ def stop(self):
         logger.info("Master stopped")
 
     def create_pod_manager_if_needed(self, args):
-        # TODO: set None if args.need_pod_manager is False.
-        self.pod_manager = PodManager(args)
+        if args.need_task_manager:
```

Comment:
Is it `args.need_pod_manager` not `args.need_task_manager`?

Reply:
Good catch, updated.